README.md

```diff
@@ -21,3 +21,21 @@ params = {
 }
 response = requests.post(url, params = params)
 ```
+
+### Verify OTP
+Use the below endpoint to verify OTP code sent to a mobile number.
+```
+https://ym6gqg7jui.execute-api.ap-south-1.amazonaws.com/dev/verifyotp
```

Comment:
we should not expose the API link here. Instead, we can use it as an environment variable.
Then, you'll also need to add a section explaining what all the environment variables mean (currently, it would just be this one env variable)

Reply:
I'm confused about this. Are we not providing this OTP auth as a service? If we aren't, then why are we explaining how to use the API?